cdn/cdn.go

```diff
@@ -0,0 +1,38 @@
+package cdn
+
+import (
+	"context"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/cdn/mgmt/cdn"

```

Comment:
Ah, something else. We are not using profiles on the samples . We are locking to a version

Reply:
Fixed in 2106698